manifests/a/Autodesk/ShotgridRV/2022.2.0/Autodesk.ShotgridRV.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Autodesk.ShotgridRV
+PackageVersion: 2022.2.0

```

Comment:
```suggestion
PackageVersion: 22.2.0
```

Reply:
`2022.2.0` will download 700 MB+ unlimited amount of times due to version mismatch which is not what we want as this will waste the user's internet bandwidth.

This will be solved in future versions of WinGet so it's best to keep it as `22.2.0` to match Control Panel rather than to keep it as `2022.2.0`